main.c

```diff
@@ -191,8 +193,68 @@ static cairo_status_t write_func(void *data, const unsigned char *buf,
 	return CAIRO_STATUS_SUCCESS;
 }
 
+void default_filename(char *filename, size_t n, int filetype) {
+	time_t time_epoch = time(NULL);
+	struct tm *time = localtime(&time_epoch);
+	if (time == NULL) {
+		perror("localtime");
+		exit(EXIT_FAILURE);
+	}
+
+	char *format_str;
+	switch (filetype) {
```

Comment:
I think we're missing PPM?

Reply:
Was working off an old branch. Fixed now.